charts/gha-runner-scale-set/values.yaml

```diff
@@ -44,6 +44,36 @@ githubConfigSecret:
 
 # runnerGroup: "default"
 
+## Custom CA certificates can be provided by bundling them in a configmap and
+## referring it in githubServerTLS.certConfigMapRef, for example:
+#
+# githubServerTLS:
+#   certConfigMapRef: your-configmap-name
```

Comment:
do we need a nested map? or just `githubServerTLSCertConfigMapRef: your-configmap-name`

Reply:
To be honest I've used the existing spec for all this without giving it enough though, you have a good point for sure. Much like proxy it's a discussion of convenience (no nesting) vs leaving the door open to other ways to provide certificates? If we want to go the way we went for proxy then it should probably stay as is, but open for discussion. What do you think? @actions/actions-runtime any thoughts?

Reply:
I am fine with what you have (nested), the only other thing I can think of for the future is to support SSL Client cert...